autoPlay.user.js

```diff
@@ -1251,13 +1292,18 @@ function tryUsingAbility(abilityId) {
 	return true;
 }
 
-function tryUsingItem(itemId) {
-	if (!canUseItem(itemId)) {
-		return false;
-	}
-
-	triggerItem(itemId);
-	return true;
+function tryUsingItem(itemId, checkInLane) {
+    if (!canUseItem(itemId)) {
+        return false;
+    }
+    if (typeof checkInLane === "undefined") {
```

Comment:
Yeah, sure. I thought it was more of a good practice to state that checkInLane is optional and defaults to false.
I can remove it if you wish though.